src/platform/site-wide/sass/modules/_m-medallia.scss

```diff
@@ -11,3 +11,7 @@ button {
     }
   }
 }
+
+.kampyle_feedback-button.kampyle_bottom-right .kampyle_triangle {
+  border-bottom: 81px solid #fdb81e !important;
```

Comment:
The original style is added directly to a `<style>` node which AFAIK requires an `!important` attribute for an imported stylesheet to override.

Reply:
Yeah, if that `<style>` is added by the 3rd party library there isn't much we can do here. I do wonder if it's possible to just do an override of just the `border-color` or `border-bottom-color` here.

Reply:
good idea- updated.